### PR TITLE
benchmark: fix `-h` parameter to exit after printing help

### PR DESCRIPTION
**Pull request by yangdihangN**

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @yangdihangN isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by yangdihangN** [comment]
11

**Comment by yangdihangN** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            
        
    
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                ivanyangtt
                            
                            
                                
                            
                    
                        
                            
                                    ivanyangtt@zju.edu.cn
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 3/20/2019 15:20，The Linux Foundation<notifications@github.com> wrote： 


Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.
After the following items are addressed, please respond with a new comment here, comment and the automated system will re-verify.

User @yangdihangN isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

Regards,
CLA GitHub bot

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by yangdihangN** [comment]
11

File: benchmark/run_bench.sh [hunk 1]
```diff
@@ -163,6 +163,7 @@ while [ $# -gt 0 ]; do
       echo "-req              req size in bytes, default value is 1"
       echo "-resp             resp size in bytes, default value is 1"
       echo "-rpc_type         valid values are unary|streaming, default is unary"
+      exit 1
```
Comment (dfawley) [inline benchmark/run_bench.sh]: It would be more natural to `exit 0` in this case, since there was no error.

Comment (yangdihangN) [reply benchmark/run_bench.sh]: ok，I have updated the code

**Comment by dfawley** [comment]
Great, thanks again!

**Pull request merged**